webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,18 @@ function BugForm() {
     }, this));
   };
 
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand", function() {
+      this.baseScrollHeight = this.scrollHeight;
+    })
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      var minRows = 2;
+      this.rows = minRows;
+      var rows = Math.ceil((this.scrollHeight - this.baseScrollHeight) / 17);
```

Comment:
Can you leave a comment explaining where the number 17 comes from? It will be confusing to come back to later. :)